assets/scss/_variables_project.scss

```diff
@@ -49,3 +49,13 @@ h2, h3 {
 {
   clear: both;
 }
+
+
+.td-sidebar-nav__section-title a {
+    color: #fff;
+}
+
+.td-sidebar-nav__section ul {
+    padding: 0;
+    margin: 0 0 0 0.5em;
+}
```

Comment:
These changes aren't showing for me locally. The theme style is being considered ahead of this project (S)CSS file. One way to get around this is to make these `!important`, which I recommend hesitantly! Another option is to have a project file for `_sidebar-tree.scss`, but you'd have to copy everything, not just add to it like we do with the `_variables_project.scss` file -- which seems worse than the two `!important` here. 

These changes also have unintended consequences:
- the titles are white on white -- seems as though you want that to be `color: $secondary` to be orange as per above.
- the margin on the `ul`s pushes all sections in, meaning that the top level page (why-the-cht) doesn't line up with it.

I'll see what we can do to fix this and we can revisit together.